ReactWindows/ReactNative.Shared/Modules/Core/Timing.cs

```diff
@@ -3,7 +3,11 @@
 using System;
 using System.Collections.Generic;
 using System.Threading;
+#if WINDOWS_UWP
 using Windows.UI.Xaml.Media;
```

Comment:
It does appear however that my `NET46` constant got unset by one of the merges, so I'm adding that back in.